### ISSUE TITLE
Summer/Fall 2021 publication thread

### ISSUE THREAD

**Issue by ctschroeder** [CopticScriptorium/corpora#77]
### Proposed timeline (as of July 9)

- July 25: Freeze *list* of corpora and documents
- August 2: Documents made available for review on Gitdox
- August: review and prep for publication
- Data freeze/version date TBD (? @amir-zeldes do you have a preference?)

### Corpora (@ctschroeder will link to issues with checklists in corpus repos soon)

- [x] God Says through Those Who Are His ([see repo issue for details](https://github.com/CopticScriptorium/shenoute-discourses5-dev/issues/50); *do not check off* until issue is complete)
- [x] A22 ([see repo issue for details](https://github.com/CopticScriptorium/shenoute-a22-dev/issues/52); *do not check off* until issue is complete)
- [x] You too oh prince of Evil ([see repo issue for details](https://github.com/CopticScriptorium/shenoute-discourses4-dev/issues/8); *do not check off* until issue is complete)
- [x] Marcion ([see repo issue for details](https://github.com/CopticScriptorium/budge-dev/issues/11); *do not check off* until issue is complete)
  - [x] celestinus.encomium
  - [x] flavianus.encomium
- [x] AP ([see repo issue for details](https://github.com/CopticScriptorium/ap-dev/issues/90); *do not check off* until issue is complete)
- [ ] Gospel Thomas (at least part 1) (@babelalexandria)

### Possible corpora
- [ ] shenoute.night (see [questions that need resolution before going in the queue](https://github.com/CopticScriptorium/shenoute-discourses4-dev/issues/9))

**Comment by ctschroeder** [comment]
Hey everyone. It's time to get ready for the next release.  Please check the timeline proposed above and also if your corpora are listed correctly. As usual I will make more detailed checklists in issues in specific repositories for each corpus soon.

**Comment by amir-zeldes** [comment]
I think for Marcion we have the two encomiums (Lance is doing Celestinus and I've been doing Flavianus). We also have shenoute.night in the pipeline, it's not long so should be easy. Am I forgetting something @lancealanmartin ?

**Comment by lancealanmartin** [comment]
I think that is all of marcion. I should finish the segmentation for celestinus.03 and shenoute.night soon. Mitchell said that he is nearly finished adding the translation for celestinus.01.

**Comment by ctschroeder** [comment]
Is this version 4.2.0?

**Comment by amir-zeldes** [comment]
Yes, I think that's the next number (third digit is for minor corrections/bug fixes, and this is not a major new version I think)

**Comment by ctschroeder** [comment]
@amir-zeldes AP are done. I set the new ones to "entities" status rather than "to_publish" because I know you're adding entities to everything. You can get started on this corpus for publication.

**Comment by amir-zeldes** [comment]
OK, I did a pass on those three AP and added entities+identities, should be good to go (just need version date, are we aiming for 2021-08-31?)

**Comment by ctschroeder** [comment]
Can we maybe say 9/3 or 9/7? Start of k-12 and OU semester kinda broke me this last week!!

**Comment by amir-zeldes** [comment]
Sure, no rush. I'll set version_date automatically for everything once it's all squared away.

**Comment by ctschroeder** [comment]
A22 is ready except for identities and entities

**Comment by ctschroeder** [comment]
I think I've done everything I can for all except Prince and Thomas. Prince I'll finish tonight or tomorrow. I will check in with Paul on Thomas. But for everything else I believe the ball is in @amir-zeldes & @lancealanmartin 's court for final touches and publication.

**Comment by amir-zeldes** [comment]
OK, I've checked entities for A22, but I'm not sure about one thing - is the Pishoy in a22.YA309-310 this Pishoy?

https://en.wikipedia.org/wiki/Pishoy

I also can't find an article for Atripe which is too bad. If either of you feel like adding it to Wikipedia that would be great! For now it's just "pass", but we can always change it in an upcoming release if/when it gets added as an entry.

**Comment by ctschroeder** [comment]
Oh no it’s the Red Monastery Pshoi https://en.wikipedia.org/wiki/Red_Monastery <https://en.wikipedia.org/wiki/Red_Monastery> but it appears there’s no entry. 

Atripe does have an entry, as Athribis https://en.wikipedia.org/wiki/Athribis_(Upper_Egypt) <https://en.wikipedia.org/wiki/Athribis_(Upper_Egypt)>. Entry exists but is stunningly short.

**Comment by ctschroeder** [comment]
Sorry yes everything in A22 is ready, I changed the designations in Gitdox.

I created a draft article for Pshoi but it is waiting for approval. https://en.wikipedia.org/wiki/Draft:Pshoi

**Comment by amir-zeldes** [comment]
That's amazing, thanks!! So glad we are also contributing to Wikipedia :) Is it a problem that the title is very similar to the article on Pishoy? Should it be called "Pshoi (monastic leader)" or something?

**Comment by ctschroeder** [comment]
Yeah that’s a good idea. I can’t move it though. Either because it’s a draft or because my account isn’t auto confirmed? If you can move it to a page with that title, that would be great.

**Comment by amir-zeldes** [comment]
No, I think it's locked to you right now, but let's see what the mods say.

For Prince it looks like it's a bit different from our segmentation practices here and there, how close to ready is it?

**Comment by ctschroeder** [comment]
I’m reviewing Prince now. I’m on the second doc. First one should be in pretty good shape. Am trying to get it done tonight. Will ping you as soon as I’m done

**Comment by amir-zeldes** [comment]
OK, here is a quick status report of where we are AFAIK:

### Old corpora

The following are slated for re-release due to corrections and/or new documents (1+ documents in `review` or `to_publish` state with edits since last release):

  * AP
  * Besa
  * johannes
  * onnophrius
  * pisentius
  * victor
  * athanasius
  * timothy
  * chrysostom
  * ruth
  * mark
  * a22
  * those

I have just gone over all of these corpora, and they are now all validating (all greens), all with updated version number and date (still tentatively today, could be changed en masse), **except** for the collection metadata question in shenoute.those. NLP is running to reparse them with the latest brand new parser from Luke.

### New corpora

The following are all done, except for waiting for word on **shenoute.prince**:

  * celestinus
  * flavianus
  * shenoure.night
  * shenoute.prince?

Not currently included: Thomas (which it sounds like might need to wait for next round?)

### OT

Complete re-parse following QA from @lancealanmartin just completed, ready for release with entity linking


### TODO

So basically everything is in `to_publish` state (or `published`) and valid, and we need final word on:

  * shenoute.prince part 2
  * shenoute.those.GL269-270 collection field (looks like it was already published with 'unknown' last time)
  * thomas (presumably not included?)

**Comment by ctschroeder** [comment]
Prince is not done but working on it more tonight.  Your assessment of everything else looks right to me

**Comment by ctschroeder** [comment]
Re collection metadata: I thought I sent an email that it might be Egypte but I’m not 100%. Not at my laptop but if we’ve left that field blank before we can leave it blank now instead of “unknown”.

**Comment by ctschroeder** [comment]
@amir-zeldes before we close this thread and open a new one for spring, can we change the GitDox "status" of everything that's been published? Thank you so much!!!

**Comment by amir-zeldes** [comment]
Done!
